src/App.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import './App.scss';
+import { MoviesList } from './components/MoviesList';
+import moviesFromServer from './api/movies.json';
+import { SearchMovie } from './components/SeacrhMovie';
+
+export class App extends Component {
+  state = {
+    query: '',
+  };
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value,
+    });
+  }
```

Comment:
```suggestion
    const { value } = event.target;

    this.setState({
      query: value,
    });
  }
```
Why do you need 'name' here? 'value' would be enough